packages/smartcontract/contracts/JanusNameService.sol

```diff
@@ -26,21 +27,22 @@ contract JanusNameService {
         string storageHash;
         uint expires;
         address owner;
+        uint ownerIndex;
     }
 
-    TopDomain[] private topDomains;
-    mapping(bytes32 => uint) private topDomainsHashMap;
-    mapping(address => uint[]) private topDomainsOwnerMap;
+    TopDomain[] public topDomains;
+    mapping(bytes32 => uint) public topDomainsHashMap;
+    mapping(address => uint[]) public topDomainsOwnerMap;
 
-    Domain[] private domains;
```

Comment:
why did you change the domains array and mappings to public?

Reply:
Same reason as above.

Reply:
yeah, i forgot about the tests.
Make sense.